sql/create_table_sap_ambassadors.sql

```diff
@@ -0,0 +1,64 @@
+-- phpMyAdmin SQL Dump
+-- version 4.4.8
+-- http://www.phpmyadmin.net
+--
+-- Host: localhost
+-- Generation Time: Dec 05, 2015 at 04:24 PM
+-- Server version: 5.5.46-0ubuntu0.14.04.2
+-- PHP Version: 5.5.9-1ubuntu4.14
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+
+--
+-- Database: `felicity`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `sap_ambassadors`
+--
+
+CREATE TABLE IF NOT EXISTS `sap_ambassadors` (
+  `id` int(10) unsigned NOT NULL,
+  `name` varchar(64) NOT NULL,
+  `email` varchar(64) NOT NULL,
```

Comment:
Hmm, good question. I asked Jeevan, and he told me that we'll take care of actual logging in once applications are reviewed and accepted.